### PR TITLE
Move S3N UFS as is from core Alluxio

### PR DESCRIPTION
**Pull request by madanadit**
No changes made to project (will not build). I will create a follow up with pom changes

### CODE REVIEW

**Comment by madanadit** [comment]
@aaudiber Can you take a look?

File: underfs/s3n/pom.xml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                  <excludes>
+                    <exclude>LICENSE</exclude>
+                    <exclude>META-INF/LICENSE</exclude>
+                    <exclude>META-INF/*.SF</exclude>
+                    <exclude>META-INF/*.DSA</exclude>
+                    <exclude>META-INF/*.RSA</exclude>
+                  </excludes>
+                </filter>
+              </filters>
+              <relocations>
```
Comment (apc999) [inline underfs/s3n/pom.xml]: do we still need to relocate?

Comment (madanadit) [inline underfs/s3n/pom.xml]: I was planning on fixing the pom in a separate PR to make the review easy and do a blind copy first

Comment (apc999) [inline underfs/s3n/pom.xml]: SG

**Pull request merged**